src/WordPress/Hooks.php

```diff
@@ -250,4 +252,11 @@ public function http2ServerPushInit()
     {
         HTTP2ServerPush::init();
     }
+
+    public function getCloudflareRequestJSON()
+    {
+        if ($_GET['action'] === 'cloudflare_proxy') {
```

Comment:
When a request is PATCH or DELETE does `$_GET['action']` still be occupied with `cloudflare_proxy`?